cmd/utils/flags.go

```diff
@@ -1874,11 +1874,11 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *ethconfig.Config) {
 				if !genesis.Config.TerminalTotalDifficultyPassed {
 					Fatalf("Bad developer-mode genesis configuration: terminalTotalDifficultyPassed must be true in developer mode")
 				}
-				if genesis.Config.TerminalTotalDifficulty == nil {
-					Fatalf("Bad developer-mode genesis configuration: terminalTotalDifficulty must be specified.")
+				if genesis.Config.TerminalTotalDifficulty.Cmp(big.NewInt(0)) != 0 {
+					Fatalf("Bad developer-mode genesis configuration: terminalTotalDifficulty must be 0 in developer mode")
```

Comment:
Both of these error messages can just end with "must be 0".  we've already stated that it's a dev-mode issue in the error.

Reply:
Sounds good. I was copying the diction in the error above it for `terminalTotalDifficultyPassed`. I can remove that.